inference-engine/ie_bridges/python/install/wheel/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2018-2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set(WHEEL_PACKAGE_NAME "openvino" CACHE STRING "Name of the package")
+set(WHEEL_LICENCE_TYPE "Proprietary - Intel" CACHE STRING "License for the package")
```

Comment:
Why not Apache 2.0?

Reply:
That is a good question, I defined the same license version as we use in the installer. I will change this type to Apache 2.0 as the default value and double-check if we can use it for distribution.

Reply:
Fixed, defined Apache 2.0 license by default.